### DISCUSSION TITLE
custom satellite magnitude

### DISCUSSION THREAD

**Discussion by axd1967** [Stellarium/stellarium#1989] [Q&A] [answered]
I tried to specify a magnitude for Tianhe, but nothing happens.

Am I mistaken, or is the user config copy `modules/Satellites/qs.mag` never actually used? It is copied when the file is missing, but editing that file has no effect; instead, Stellarium would still work with a Qt-provided resource file that is compiled at build time? 
https://github.com/Stellarium/stellarium/blob/6bb6746489336b1e0fb76bc1e5192eda448c62ff/plugins/Satellites/src/Satellites.cpp#L1928

**Comment by gzotti** [comment, marked as answer]
There is currently no indication that a user copy of this file would be allowed or intended. It seems there may have been one some long time ago, as I also have, err, had one, from around 2014.

**Comment by axd1967** [comment]
I guess I mixed up with the possibility to manually change comet magnitudes.

And it indeed no longer shows up when creating a pristine Stellarium profile; I must have dragged it along for a long while.

It might be interesting to be able to manually edit the magnitude field in the sat dialog, eg for TIANHE.

But where is the original data coming from?
